dora/core/common/src/main/java/alluxio/conf/PropertyKey.java

```diff
@@ -4275,34 +4175,6 @@ public String toString() {
           .setConsistencyCheckLevel(ConsistencyCheckLevel.WARN)
           .setScope(Scope.WORKER)
           .build();
-  public static final PropertyKey WORKER_BLOCK_ANNOTATOR_CLASS =
```

Comment:
The `WORKER_EVICTOR_CLASS` above can also be removed?

Reply:
can be removed, will do